src/app/issue/summary/summary.component.ts

```diff
@@ -91,10 +94,7 @@ export class SummaryComponent extends SubscriptionWrapper implements OnInit {
         this.issueId,
         this.listOfRequirements
       ),
-      (data) =>
-        this.router.navigateByUrl(
-          `${this.companyId}/projects/${this.projectId}/issues/${this.issueId}`
-        )
+      (data) => (this.summaryCreated = true)
```

Comment:
Du könntest auch in currentIssue.issueDetail.requirementsSummaryCreated speichern